wqflask/wqflask/correlation/pre_computes.py

```diff
@@ -167,3 +168,49 @@ def get_datasets_data(base_dataset, target_dataset_data):
         samples_fetched, base_traits_data)
 
     return (target_results, base_results)
+
+
+def fetch_text_file(dataset_name, conn, text_dir=TEXTDIR):
+    """fetch textfiles with strain vals if exists"""
+
+    with conn.cursor() as cursor:
+        query = 'SELECT Id, FullName FROM ProbeSetFreeze WHERE Name = "%s"' % dataset_name
+        cursor.execute(query)
```

Comment:
**tip:** Notice you only use query once.  Thus just pass it directly to the execute method.

**issue:** Please use %s like so:
```suggestion
        cursor.execute("SELECT Id, FullName FROM ProbeSetFreeze WHERE Name = %s", (dataset_name,))
```

This sanitizes the string for you and prevents and sql injection from any malicious source.